doc/source/whatsnew/v0.21.0.txt

```diff
@@ -231,7 +231,7 @@ Previously, :func:`to_datetime` did not localize datetime ``Series`` data when `
 
      pd.to_datetime(s, utc=True)
 
-Additionally, DataFrames with datetime columns returned by :func:`read_sql` will also be localized to UTC only if the original SQL columns were timezone aware datetime columns.
+Additionally, DataFrames with datetime columns returned by :func:`read_sql_table` and :func:`read_sql_query` with the `parse_dates` argument specified will also be localized to UTC only if the original SQL columns were timezone aware datetime columns.
```

Comment:
for read_sql_query with parse_dates this was already the case? 
If so, I would maybe leave it out, as the interpretation can be dubious: you can read it as "(read_sql_table and read_sql_query) with parse_dates" instead of "read_sql_table and (read_sql_query with parse_dates)"